kqueen/config/utils.py

```diff
@@ -42,6 +43,8 @@ def apply_env_changes(config, prefix='KQUEEN_'):
     for name, value in os.environ.items():
         if name.startswith(prefix):
             config_key_name = name[len(prefix):]
+            if re.search('(?i)true|(?i)false', value):
```

Comment:
I still think that the rest of this PR could be entirely avoided if you tested somehow that the env variable is supposed to be list/dict and used `json.load` on the next line here. But it is just a suggestion, I don't have any other solution, so I will approve if you just fix the stuff from other comments.

Reply:
it supposed, but IMO and for my experience, there is really bad case to define dicts in env-vars, coz there a lot of dependencies, for example if u compile docker stack arch, using reclass+salt+yml, smth can went wrong with dict syntax, so i prefer do not use that, but we can file a us and investigate it in future